src/MountConfig.js

```diff
@@ -24,6 +24,55 @@ class MountConfig extends SchemaDerivedConfig {
         '^/mountpoints$': NamedPairHandler('path', 'url'),
       },
     });
+    this._sanitized = null;
+  }
+
+  get mountpoints() {
+    if (!this._sanitized) {
+      this._sanitized = (this._content.mountpoints || []).map((mp) => {
+        const m = { ...mp };
+        if (!m.path.endsWith('/')) {
+          m.path += '/';
+        }
+        if (!m.type && m.url) {
+          if (!m.id && m.url.startsWith('https://drive.google.com/')) {
+            m.type = 'google';
+            m.id = m.url.split('/').pop();
+          } else if (/https:\/\/.*\.sharepoint\.com/.test(m.url) || m.url.startsWith('https://1drv.ms/')) {
+            m.type = 'onedrive';
+          } else {
+            m.type = 'unknown';
+          }
+        }
+        return m;
+      });
+      // sort by path, descending
+      this._sanitized.sort((m1, m2) => (m2.path.localeCompare(m1.path)));
```

Comment:
Not sure if sorting is a good idea, why shouldn't we just take the first match?

Reply:
> Not sure if sorting is a good idea, why shouldn't we just take the first match?

it's the old map vs list problem. probably better to change the schema to support a list of objects, similar to the strains:

eg:
```
mountpoints:
  - path: /ms/docs
    url: https://adobe.sharepoint.com/sites/docs
  - path: /ms
    url: https://adobe.sharepoint.com/sites/TheBlog/Shared%20Documents/theblog
  - path: /gd
    url: https://drive.google.com/drive/u/0/folders/123456789
  - path: /foo
    url: https://localhost:4502
```